CFC_DataCollector/recommender/common.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import division
+import urllib2
+import xml.etree.ElementTree as ET
+from get_database import *
+from datetime import datetime, timedelta
+import trip
+from trip import E_Mission_Trip
+import random
+import json
+import jsonpickle
+
+def get_uuid_list():
+	uuid_list = [ ]
+	db = get_section_db()
+	for x in db.find():
+   		uuid_list.append(x['_id'])
+	return uuid_list
+
+def insert_into_pdb(pdb, my_id, new_perturbed_trip):
```

Comment:
@joshzarrabi, why are these in `common` and not in the alternatives module? Are they invoked from anywhere other than the `alternatives` module? If you don't want to put `perturbed` functions into the `alternatives` module, you could make a new `perturbed` module.